raw.js

```diff
@@ -0,0 +1,27 @@
+var bel = require('.')
+
+function rawCreateElement (tag) {
+  if (typeof window !== 'undefined') {
+    return browser()
+  } else {
+    return server()
+  }
+
+  function browser () {
+    var el = bel`<div></div>`
+    el.innerHTML = tag
+    return toArray(el.childNodes)
+  }
+
+  function server () {
+    var wrapper = String(tag)
+    wrapper.__encoded = true
```

Comment:
What does this do again?

Reply:
In the browser we can use an unescaped string by using `.innerHTML`, and unwrapping the children into an array to avoid unwanted element nesting. `pelo` doesn’t have access to the DOM, but you can skip escaping the value by setting `__encoded` as seen here: https://github.com/shuhei/pelo/blob/0faa1e56a816312a7bbedb970be436f617b00fb3/index.js#L27